docker/kernel/httploader/httploader/falcoloader/loader.go

```diff
@@ -0,0 +1,148 @@
+package falcoloader
```

Comment:
I'm confused by the packaging.
I see a tree: `falco/docker/kernel/httploader/httploader/falcoloader` and then the `loader.go` file.

We can simplify that to: `falco/docker/kernel/httploader/pkg/falcoloader` to contain the `loader.go` file and `falco/docker/kernel/httploader/` to contain `main.go` and `Dockerfile`.

Also I'm not sure we want to have code inside the `docker/kernel` folder, I expect the `docker` folder to contain only dockerfiles, what do you think if we move this code up in it's own directory like `falco/httploader` ?

Reply:
We need to setup go modules for this